style/styles_shop.css

```diff
@@ -0,0 +1,158 @@
+/* задание фона */
+.shop {
+  background-image: url("../pic/shop_back.png");
+  font-family: "Gotham Pro Bold", sans-serif;
+  height: 280px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.shop__title {
+  color: #fff;
+  font-size: 30px;
+  width: 195px;
+  margin: 0;
+  padding: 16px 59px;
+}
+
+/* создание ссылки со стрелкой */
+.shop__link {
+  color: #fff;
+  font-size: 12px;
+  padding-left: 59px;
+  text-decoration: none;
+}
+
+.shop__link::after {
+  content: url("../pic/Arrow.png");
+  padding-left: 8px;
+}
+
+/* верстка для 1024рх */
+@media (max-width: 1024px) {
+  .shop {
+    background-image: url("../pic/shop_back1024.png");
+    height: 254px;
+  }
+
+  .shop__title {
+    padding: 10px 54px;
+  }
+
+  .shop__link {
+    padding-left: 54px;
+  }
+
+  .shop__link::after {
+    content: url("../pic/Arrow1024.png");
+    padding-left: 11px;
+  }
+}
+
+/* верстка для 768рх */
+@media (max-width: 768px) {
+  .shop {
+    background-image: url("../pic/shop_back768.png");
+    height: 415px;
+  }
+
+  .shop__title {
+    font-size: 50px;
+    line-height: 49px;
+    width: 355px;
+    padding: 42px 45px;
+  }
+
+  .shop__link {
+    font-size: 20px;
+    padding-left: 45px;
+  }
+
+  .shop__link::after {
+    content: url("../pic/Arrow768.png");
+    padding-left: 9px;
+  }
+}
+
+/* промежуточная верстка 630px */
+@media (max-width: 630px) {
+  .shop {
+    background-image: url("../pic/shop_back630.png");
+    height: 340px;
+  }
+
+  .shop__title {
+    font-size: 35px;
+    line-height: 30px;
+    padding: 29px 0 14px 48px;
+  }
+}
+
+/* прожеуточная верстка 500px */
+@media (max-width: 500px) {
+  .shop {
+    background-image: url("../pic/shop_back500.png");
+    height: 270px;
+  }
+
+  .shop__title {
+    font-size: 25px;
+    line-height: 25px;
+    padding: 29px 0 14px 48px;
+    width: 165px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5525083)